wrangler-service/src/main/java/co/cask/wrangler/service/spanner/SpannerResponse.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.wrangler.service.spanner;
+
+import java.net.HttpURLConnection;
+import java.util.List;
+
+/**
+ * Response sent by spanner service methods
+ * @param <T>
+ */
+public class SpannerResponse<T> {
```

Comment:
this doesn't seem specific to Spanner. Could it be used as the response for other API calls? If so, lets rename it.

Reply:
yes, its not specific to Spanner.